### PR TITLE
** DO NOT MERGE ** Compute Resource Consolidation pattern update - Removed obsolete example on the doc, and pattern implementation examples

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

**Comment by PRMerger9** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/patterns/compute-resource-consolidation.md [hunk 1]
```diff
@@ -79,159 +79,14 @@ Use this pattern for tasks that are not cost effective if they run in their own
...
-
-    // If any of the inner exceptions in the aggregate exception
-    // are not cancellation exceptions then rethrow the exception.
-    ex.Handle(innerEx => (innerEx is OperationCanceledException));
-  }
-}
-```
+- Azure App Service and Azure Functions: Deploy shared App Service plans, which represent the hosting server infrastructure. One or more apps can be configured to run on the same computing resources (or in the same App Service plan).
+- Azure Container Apps: Deploy container apps to the same shared environments; especially in situations when you need to manage related services or you need to deploy different applications to the same virtual network.
+- Azure Kubernetes Service (AKS): Deploy shared pods, with a multitenancy-aware application.
```
Comment (magrande) [inline docs/patterns/compute-resource-consolidation.md:88]: @ckittel  what's your suggestion here ?

"Azure Kubernetes Service (AKS): Deploy shared pods, with a multitenancy-aware application."

Comment (ckittel) [reply docs/patterns/compute-resource-consolidation.md:88]: Azure Kubernetes Service (AKS): AKS is a container-based hosting infrastructure in which multiple applications or application components can be configured to run co-located on the same computing resources (nodes), grouped by computational requirements such as CPU or memory needs (node pools).

What about something like this?

**Comment by PRMerger15** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/patterns/compute-resource-consolidation.md (hunk 1 above)
Comment (magrande) [reply docs/patterns/compute-resource-consolidation.md]: Perfect ! Done.

File: docs/patterns/compute-resource-consolidation.md [hunk 2]
```diff
@@ -21,21 +21,21 @@ Consolidate multiple tasks or operations into a single computational unit. This
 
 ## Context and problem
```
Comment (ckittel) [inline docs/patterns/compute-resource-consolidation.md:22]: Can you update the metadata at the top of this file?  Updating the date if you've reviewed the content.

File: docs/patterns/compute-resource-consolidation.md [hunk 3]
```diff
@@ -21,21 +21,21 @@ Consolidate multiple tasks or operations into a single computational unit. This
...
+A cloud application often implements a variety of operations. In some solutions it makes sense to follow the design principle of separation of concerns initially, and divide these operations into separate computational units that are hosted and deployed individually (for example, as separate App Service web apps, separate Virtual Machines). However, although this strategy can help simplify the logical design of the solution, deploying a large number of computational units as part of the same application can increase runtime hosting costs and make management of the system more complex.
```
Comment (ckittel) [inline docs/patterns/compute-resource-consolidation.md:24]:
```suggestion
A cloud application often implements a variety of operations. In some solutions it makes sense to follow the design principle of separation of concerns initially, and divide these operations into separate computational units that are hosted and deployed individually (for example, as separate App Service web apps or separate Virtual Machines). However, although this strategy can help simplify the logical design of the solution, deploying a large number of computational units as part of the same application can increase runtime hosting costs and make management of the system more complex.
```

File: docs/patterns/compute-resource-consolidation.md [hunk 4]
```diff
@@ -21,21 +21,21 @@ Consolidate multiple tasks or operations into a single computational unit. This
...
 
 ![Running tasks in a cloud environment using a set of dedicated computational units](./_images/compute-resource-consolidation-diagram.png)
 
 Each computational unit consumes chargeable resources, even when it's idle or lightly used. Therefore, this isn't always the most cost-effective solution.
 
-In Azure, this concern applies to roles in a Cloud Service, App Services, and Virtual Machines. These items run in their own virtual environment. Running a collection of separate roles, websites, or virtual machines that are designed to perform a set of well-defined operations, but that need to communicate and cooperate as part of a single solution, can be an inefficient use of resources.
+In Azure, this concern applies to App Services, Container Apps and Virtual Machines. These items run in their own virtual environment. Running a collection of separate websites, microservices, or virtual machines that are designed to perform a set of well-defined operations, but that need to communicate and cooperate as part of a single solution, can be an inefficient use of resources.
```
Comment (ckittel) [inline docs/patterns/compute-resource-consolidation.md:32]:
```suggestion
In Azure, this concern applies to App Services, Container Apps, and Virtual Machines. These items run in their own environment. Running a collection of separate websites, microservices, or virtual machines that are designed to perform a set of well-defined operations, but that need to communicate and cooperate as part of a single solution, can be an inefficient use of resources.
```

**Review by ckittel — CHANGES_REQUESTED** [review]
Just a couple suggestions. Thanks!

**Comment by PRMerger12** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/patterns/compute-resource-consolidation.md (hunk 2 above)
Comment (magrande) [reply docs/patterns/compute-resource-consolidation.md:22]: Date updated

**Comment by PRMerger5** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger3** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/patterns/compute-resource-consolidation.md [hunk 5]
```diff
@@ -79,159 +79,14 @@ Use this pattern for tasks that are not cost effective if they run in their own
...
-    // If any of the inner exceptions in the aggregate exception
-    // are not cancellation exceptions then rethrow the exception.
-    ex.Handle(innerEx => (innerEx is OperationCanceledException));
-  }
-}
-```
+- Azure App Service and Azure Functions: Deploy shared App Service plans, which represent the hosting server infrastructure. One or more apps can be configured to run on the same computing resources (or in the same App Service plan).
+- Azure Container Apps: Deploy container apps to the same shared environments; especially in situations when you need to manage related services or you need to deploy different applications to the same virtual network.
+- Azure Kubernetes Service (AKS): AKS is a container-based hosting infrastructure in which multiple applications or application components can be configured to run co-located on the same computing resources (nodes), grouped by computational requirements such as CPU or memory needs (node pools).
+- Virtual machines: Deploy a single set of virtual machines for all tenants to use, that way the management costs are shared accross the tenants.
```
[the remark this answers is not in the document]
Comment (magrande) [reply docs/patterns/compute-resource-consolidation.md]: Done

File: docs/patterns/compute-resource-consolidation.md [hunk 6]
```diff
@@ -79,159 +79,14 @@ Use this pattern for tasks that are not cost effective if they run in their own
...
-    ex.Handle(innerEx => (innerEx is OperationCanceledException));
-  }
-}
-```
+- Azure App Service and Azure Functions: Deploy shared App Service plans, which represent the hosting server infrastructure. One or more apps can be configured to run on the same computing resources (or in the same App Service plan).
+- Azure Container Apps: Deploy container apps to the same shared environments; especially in situations when you need to manage related services or you need to deploy different applications to the same virtual network.
+- Azure Kubernetes Service (AKS): AKS is a container-based hosting infrastructure in which multiple applications or application components can be configured to run co-located on the same computing resources (nodes), grouped by computational requirements such as CPU or memory needs (node pools).
+- Virtual machines: Deploy a single set of virtual machines for all tenants to use, that way the management costs are shared accross the tenants. Virtual Machine Scale sets is a feature that automatically allows shared resource management, load-balancing, scaling, application's highly availability with no extra cost.
```
Comment (ckittel) [inline docs/patterns/compute-resource-consolidation.md:89]:
```suggestion
- Virtual machines: Deploy a single set of virtual machines for all tenants to use, that way the management costs are shared accross the tenants. Virtual Machine Scale Sets is a feature that supports shared resource management, load-balancing, and horizontal scaling of Virtual Machines.
```

**Comment by PRMerger13** [comment]
@ckittel : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ckittel** [comment]
I'll migrate this over to the internal repo, and drop a link here when complete.

**Comment by ckittel** [comment]
Closing, PR continued @ MicrosoftDocs/architecture-center-pr#5328

**Pull request closed without merging**